astropy/utils/data.py

```diff
@@ -95,7 +112,8 @@ def _is_url(string):
     # we can't just check that url.scheme is not an empty string, because
     # file paths in windows would return a non-empty scheme (e.g. e:\\
     # returns 'e').
-    return url.scheme.lower() in ['http', 'https', 'ftp', 'sftp', 'ssh', 'file']
+    return url.scheme.lower() in [
+        'http', 'https', 'ftp', 'sftp', 'ssh', 'file']
```

Comment:
In generate we tolerate lines up to 100 characters long - this is an example where I think things are less readable.